lib/ruby_lsp/server.rb

```diff
@@ -690,13 +690,13 @@ def shutdown
 
     sig { params(config_hash: T::Hash[String, T.untyped]).void }
     def perform_initial_indexing(config_hash)
-      index_core_classes
+      index_core_classes_and_modules
       index_ruby_code(config_hash)
     end
 
     sig { void }
-    def index_core_classes
-      RubyIndexer::RBSIndexer.new(@global_state.index).index_core_classes
+    def index_core_classes_and_modules
```

Comment:
The RBS definitions also cover constants, e.g. `ARGV`, that we may want to add, so perhaps we should generalize this name to something like `index_core`?

Reply:
That's fair. Since we have `index_ruby_code`, I named this `index_ruby_core` to be consistent.